configs/_base_/models/ssd300.py

```diff
@@ -42,10 +51,5 @@
         pos_weight=-1,
         neg_pos_ratio=3,
         debug=False),
-    test_cfg=dict(
-        nms_pre=1000,
-        nms=dict(type='nms', iou_threshold=0.45),
-        min_bbox_size=0,
-        score_thr=0.02,
-        max_per_img=200))
+    test_cfg=dict(nms_pre=1000))
```

Comment:
This is strange that post-processing needs to set two different positions, users may be more confused. If `nms_pre` is run after `PreNMS`, then it may be difficult to be compatible?

Reply:
It is difficult for me to move nms_pre to post_processes.